### DISCUSSION TITLE
external table support

### DISCUSSION THREAD

**Discussion by theseusyang** [risingwavelabs/risingwave#1858] [Q&A]
does risingwave support now external table  function support, please?

**Comment by liurenjie1024** [comment]
Could you provide more details? Do you mean query external data source or load external table function as udf?

**Reply by theseusyang** [reply]
like greenplum/hive external table function , can define `create external table (col1 ,col2 )` its data sources is from mysql, oracle , or other database.

**Reply by liurenjie1024** [reply]
OK, that's external data source. Currently we don't have plan on this, but we may support it in future.
